src/vos/vos_layout.h

```diff
@@ -147,8 +147,6 @@ struct vos_dtx_cmt_ent_df {
 	struct dtx_id			dce_xid;
 	/** The epoch# for the DTX. */
 	daos_epoch_t			dce_epoch;
-	/** The time of the DTX being handled on the server. */
-	daos_epoch_t			dce_handle_time;
```

Comment:
Anyway to do this while maintaining compatibility?   We are not supposed to break durable format between 2.0 and 2.2

Reply:
So we have to keep these useless fields until 2.4 or 2.6?

Reply:
Ok, I just talked to Johann.

"The path is 1. Upgrade to 2.2 and still have the bug. 2. Dmg pool upgrade and the problem is fixed (but no way back)."

So, the code will need to check the pool version and not fail on the old one (as it does now).   Then, if the new version is there, we have the bug fixed.